src/Mapping/ClassMetadata.php

```diff
@@ -2003,6 +2004,17 @@ protected function _storeAssociationMapping(AssociationMapping $assocMapping): v
      */
     public function setCustomRepositoryClass(string|null $repositoryClassName): void
     {
+        if ($repositoryClassName === null) {
```

Comment:
I'm not sure about this change. `null` is a valid value which means that for this entity, I want to use the default repository class.

Reply:
Hmmm right… let me revert this.